modules/images/dominant-color/load.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Module Name: Dominant Color
+ * Description: Adds support to store dominant-color for an image and create a placeholder background with that color.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Add the dominant color metadata to the attachment.
+ *
+ * @param array $metadata The attachment metadata.
+ * @param int   $attachment_id The attachment ID.
+ *
+ * @return array $metadata
```

Comment:
Please, do not add blank lines between params and return tags. All phpdoc blocks need to be consistent across the whole plugin. Could you please these remove blank lines in all your phpdoc blocks?
```suggestion
 * @param int   $attachment_id The attachment ID.
 * @return array $metadata
```

Reply:
Unresolving this, since the feedback hasn't been addressed yet.

Reply:
@pbearne There is still a blank line here. I know this is a nit-pick, but please address this comment (as well as the other comment from @eugene-manuilov about adding a return description.